docs/faq.md

```diff
@@ -116,6 +117,13 @@ techs made available to video.js. For example, video.js 5 includes the Flash tec
 enables the playback of FLV video where the Flash plugin is available. For more information
 on media formats see the [troubleshooting guide][troubleshooting].
 
+## Q: How can I play RTMP video in video.js?
+
+Make sure that the Flash tech is available. Then, just set the rtmp source with
```

Comment:
"...is available (RTMP is not playable on browsers without Flash including mobile)."